themes/default/content/blog/stack-readme/index.md

```diff
@@ -0,0 +1,84 @@
+
+<!--more-->
+
+There are a lot of operational activities that happen around [Pulumi Stacks](https://www.pulumi.com/docs/intro/concepts/stack), however, the information for these activities live across many places. Our [User Experience team](https://www.pulumi.com/blog/get-to-know-pulumis-ux-team) recently conducted user experience interviews with our customers and we learned users are moving across many web apps and local apps to manage their deployments- from their cloud Console, to the Pulumi Service User Interface (UI), to the CLI, and so on. By collecting resource outputs in the Pulumi Service UI, we are aiming to ease this friction and collect relevant Stack information in one place.
+
+README templates can reference Stack outputs and resource properties, such as `${outputs.foo}`. To walk you through a tangible example, a user can deploy an RDS instance and then use a variable in their README template to link to their CloudWatch dashboard to keep an eye on operational metrics. This CloudWatch dashboard link will dynamically update when deployments happen. You can use the same README template across dev, testing and production and have the correct dashboard links for each Stack.
+
+The new experience lives in the Stack page, which can be navigated to through Projects and clicking on the specific Stack you want to view the README for. Let's take a look at this new feature!
+
+![Stack READMEs in the Pulumi Console](stack-readme.png)
+
+To illustrate, let's look at the new Stack README for the Pulumi Service dev stack.
+
+```markdown
+# Pulumi Service README
+​
+[Sign in to AWS to view stack resources!](https://top-secret-url.com)
+​
+I am...
+​
+* [an on call engineer](#oncall)
+* [monitoring a deployment](#cicd)
+* [a new hire looking to onboard](#onboard)
+* [interested in querying for business insights](#insights)
+​
+-----
+​
+<a name="oncall"></a>
+## On Call Operations
+​
+### Monitor
+​
+1. [Cloudwatch Metrics](https://us-west-2.console.aws.amazon.com/cloudwatch/home?region=us-west-2#dashboards:name=${outputs.dashboardName}): Monitor holistic metrics tracking overall service health
+2. [RDS Performance Metrics](https://us-west-2.console.aws.amazon.com/rds/home?region=us-west-2#performance-insights-v20206:/resourceId/${database.databaseCluster.id}/resourceName/${outputs.rdsClusterWriterInstance}): Monitor RDS performance (wait times, top queries)
+3. [Cloudwatch Logs](https://us-west-2.console.aws.amazon.com/cloudwatch/home?region=us-west-2#logStream:group=${outputs.cloudwatchLogGroup}): Search across service logs
+```
+
+We have links to our AWS authentication tool, our CloudWatch metrics, to our production deployment documentation, and so on. The common places that we navigate to and from when managing this Stack.
+
+In order to add a README to your Pulumi Stack, you will need to do the following:
+
+- Set Stack Output named `readme` to the value of your templated Stack README file, i.e.`Pulumi.README.md`.
```

Comment:
Export a stack output named `readme` that contains your templated Stack README markdown, commonly by reading a file, i.e. `Pulumi.README.md`.